src/main/java/the_warlord/powers/GashPower.java

```diff
@@ -0,0 +1,74 @@
+package the_warlord.powers;
+
+import basemod.interfaces.CloneablePowerInterface;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.math.MathUtils;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePrefixPatch;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.ReducePowerAction;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import com.megacrit.cardcrawl.powers.watcher.VigorPower;
+import the_warlord.WarlordMod;
+
+import java.util.Random;
+
+public class GashPower extends AbstractPower implements CloneablePowerInterface {
+    public static final String POWER_ID = WarlordMod.makeID("GashPower");
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+
+    public GashPower(AbstractCreature owner, int amount) {
+        name = NAME;
+        ID = POWER_ID;
+        this.owner = owner;
+        this.amount = amount;
+        type = PowerType.BUFF;
+        updateDescription();
+        isTurnBased = true;
+        // TODO: add proper art
+        loadRegion("explosive");
+    }
+
+    @Override
+    public void updateDescription() {
+        description = DESCRIPTIONS[0];
+    }
+
+    @Override
+    public void onUseCard(AbstractCard card, UseCardAction action) {
```

Comment:
This should either use onAttacked, or check that the card played is an attack AND that the card is targetting them. I'd say onAttacked is easiest, and we'll see if we like the implications that has on multi-attack cards.